plugins/data.headers.js

```diff
@@ -284,8 +284,13 @@ exports.from_match = function (next, connection) {
         return next();
     }
 
-    const hdr_addr = (plugin.addrparser.parse(hdr_from))[0];
+    try {
+        const hdr_addr = (plugin.addrparser.parse(hdr_from))[0];
```

Comment:
I don't think this works because hdr_addr now gets block-scoped here. You need to define it with `let` outside the try and assign inside.

Reply:
> I don't think this works

Correct, or he can move the code that uses `hdr_addr` into the try block.